runtime/src/lib.rs

```diff
@@ -1040,6 +1040,29 @@ impl IsmpRouter for ModuleRouter {
     }
 }
 
+parameter_types! {
+    pub const Plonky2MaxPubsSize: u32 = 262_144;
+    pub const Plonky2MaxProofSize: u32 = 262_144;
```

Comment:
Looking at Plonky2 codebase it seems it's possible to compress the proof. You save around 10 KB of data which might be enough for us to consider

Reply:
I've brought in custom `Proof` struct to support this; you can check it out here:

https://github.com/distributed-lab/plonky2-verifier/commit/7b9efd3f68dda7fa0bb7746ee72576f7fe08b788